apps/condo/domains/meter/schema/MeterReading.test.js

```diff
@@ -1011,12 +1052,24 @@ describe('MeterReading', () => {
             const [source] = await MeterReadingSource.getAll(adminClient, { id: CALL_METER_READING_SOURCE_ID })
             const [meterReading] = await createTestMeterReading(client, meter, organization, source)
 
-            await updateTestProperty(adminClient, property.id, { deletedAt: 'true' })
+            await PropertyAPI.softDelete(adminClient, property.id)
```

Comment:
we use `API` postfix only for server-side utilities. It's test util

Reply:
fixed